css-syntax/styles.css

```diff
@@ -0,0 +1,28 @@
+body {
+color: #090302;
```

Comment:
This indentation is off. All properties in a CSS rule should be one indentation in from the selector